screenpy/resolutions/matches.py

```diff
@@ -1,10 +1,17 @@
 """Matches a string using a regex pattern."""
 
+from __future__ import annotations
+
+from re import Pattern
+from typing import TYPE_CHECKING
+
 from hamcrest import matches_regexp
-from hamcrest.core.matcher import Matcher
 
 from screenpy.pacing import beat
 
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
What is pragma? Do we use it?

Reply:
It's the comment to tell `coverage` "this section is not to be counted".  There is still an [open issue](https://github.com/nedbat/coveragepy/issues/831) to automatically ignore `if TYPE_CHECKING`.  I have been slowly adding tests based on `coverage` reports.